src/quest_manager/templates/quest_manager/quest_approval.html

```diff
@@ -72,6 +72,54 @@
 
     $('[data-toggle="tooltip"]').tooltip();
 });
+
+// all forms that has an id starting with "quick_reply"
```

Comment:
Can you add a sort-of doctstring comment here?  I know most of the other javascript doesn't but this is a function and should be documented what this block of javascript does, and how it does it.

Reply:
```
/*
  Overrides basic form function (refresh page and redirect to new url)
  To send ajax request, remove submission container, and update messages dynamically

  + if button pressed on quick reply form (quest submission) was 'approve_button' or 'return_button'
  + sends ajax request to 'quests:ajax_approve'
  + if successful removes submission container and updates messages
  */
```